mlibrary/design-system-docs
# Upgrade 11ty to v2 and 11ty-fetch

# What's new

Upgrade 11ty to the stable v2 and eleventy-fetch

What's changed
- `eleventy`
- `eleventy-fetch`

## Test
- run `npm install`
- Follow directions to add `eleventy-upgrade-help` plugin -> https://github.com/11ty/eleventy-upgrade-help
- Run plugin and check for any violations or warnings
- Poke around site
- Make changes to scss and a layout and observe changes
- Uninstall plugin and remove config from `.eleventy.js`
